src/transformStateWithClones.js

```diff
@@ -7,7 +7,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [Object.assign({}, state)];
+  let cloneObject = result[0];
+
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
```

Comment:
Move all cases to the object.

Example of usage:
`case ActionType.AddProperties:`